monitoring/alert-rules.nix

```diff
@@ -11,6 +11,11 @@ mapAttrsToList (name: opts: {
     description = opts.description;
   };
 }) {
+  node_deployment_state = {
+    condition = "node_deployment_state{older_14_d} == 1";
```

Comment:
This metric should use a timestamp instead of a match against a hard coded time range.